src/Tribe/API.php

```diff
@@ -158,13 +158,13 @@ public static function saveEventMeta( $event_id, $data, $event = null ) {
 			}
 
 			// Set sticky state for calendar view.
-			if ( isset( $data['EventShowInCalendar'] ) && $data['EventShowInCalendar'] == 'yes' && $event->menu_order != '-1' ) {
+			if ( isset( $data['EventShowInCalendar'] ) && $data['EventShowInCalendar'] == 'yes' && is_object( $event ) && $event->menu_order != '-1' ) {
```

Comment:
Let's switch this (and the other if statement) to `$event instanceof WP_Post` rather than `is_object()`